.github/workflows/pr.yml

```
@@ -52,6 +55,14 @@ jobs:
     if: ${{ needs.modified-files.outputs.app-changes == 'true' || needs.modified-files.outputs.e2e-changes == 'true' || needs.modified-files.outputs.fiftyone-changes == 'true' }}
     uses: ./.github/workflows/e2e.yml
 
+  frozen-docs:
+    runs-on: ubuntu-latest
+    needs: modified-files
+    if: ${{ needs.modified-files.outputs.docs-changes == 'true' }}
+    steps:
+      - run: echo "please target your docs changes to the voxel51-docs repository"
```

Non Blocking Nit!

Maybe we should use the link (https://github.com/voxel51/voxel51-docs) to the repo and add some additional context since this is an open-source repo. Not everyone will be aware of the docs freeze.

```suggestion
      - run: |
          echo "The /docs contributions are being migrated to https://github.com/voxel51/voxel51-docs."
          echo "Please target any documentation changes to the voxel51-docs repository."
```